From 5de2d4d68b1a4976252d860fbc0224738a3c1021 Mon Sep 17 00:00:00 2001
From: Chen Baozi <chenbaozi@phytium.com.cn>
Date: Fri, 14 Jul 2023 08:33:58 +0800
Subject: [PATCH 095/136] BACKPORT: PHYTIUM: arm64: phytium: Add support for
 Phytium SoC family

This patch adds supoort for the Phytium SoC family.

Signed-off-by: Chen Baozi <chenbaozi@phytium.com.cn>
Change-Id: Ibe06dc2e54a413a8c1257fef3aade99e6ca6480f

Link: https://gitee.com/phytium_embedded/phytium-linux-kernel/commit/154f8e5d365d57d38ad6fabc230861c9643db154
[Kexy: Resolved minor conflict in arch/arm64/Kconfig.platforms]
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/arm64/Kconfig.platforms | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 6c6d11536b42..54fc8ab89ee7 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -262,6 +262,14 @@ config ARCH_PENSANDO
 	  cores for a minimal latency/jitter datapath, and network
 	  interfaces up to 200 Gb/s.
 
+config ARCH_PHYTIUM
+	bool "Phytium SoC Family"
+	help
+	  This enables support for Phytium ARMv8 SoC family, including:
+	   - Phytium Server SoC Family
+	   - Phytium Desktop SoC Family
+	   - Phytium Embedded SoC Family
+
 config ARCH_QCOM
 	bool "Qualcomm Platforms"
 	select GPIOLIB
-- 
2.47.0

